Frontend/src/main/java/com/harmonia/client/PublicMessageClient.java

```diff
@@ -75,12 +75,12 @@ public ResponseEntity<Void> removeMessage(Long pMessageId) {
      * @param editMe The {@link PublicMessagePO} object representing the message to be edited.
      * @return The {@link PublicMessagePO} object representing the edited message.
      */
-    public PublicMessagePO editPublicMessage(PublicMessagePO editMe) {
+    public ResponseEntity<String> editPublicMessage(PublicMessagePO editMe) {
```

Comment:
This will turn your response into a String. Is the previous implementation not working?

Reply:
The backend does not return a MessagePO, it only return a string saying "Message has been edited". This cause exceptions when trying to edit messages in the server view.